tests/test.py

```diff
@@ -3406,6 +3406,33 @@ def test_generate_client_id_from_distinguished_name(self):
         readKey = self.read_key_by_client_id('distinguished_name', self.dir_with_distinguished_name_client_id.name)
         self.assertTrue(readKey)
 
+    def test_non_client_id_keys_generation(self):
```

Comment:
can we check third case with hybrid set of kinds of keys? when we generate keys that require client_id and do not, and use incorrect client_id to check that validation has been done.

Reply:
Sure, let me add that